util/fontEdit.ms

```diff
@@ -1288,6 +1308,14 @@ glyphList.checkEvents = function
 		self.scroll = mathUtil.clamp(self.scroll, 0, maxScroll)
 		self.draw
 	end if
+	if mouse.button(0) then
+		clicky = self.scroll + floor((self.top - mouse.y) / 24)
+		if clicky != self.selectedIndex then
+			self.selectedIndex = clicky
+			changeEditCharacter(selectedTab.font.chars.indexes[self.selectedIndex])
```

Comment:
No parentheses are needed on the `changeEditCharacter` call.  But this is a minor thing; overall it looks great!